substratevm/src/com.oracle.svm.configure/src/com/oracle/svm/configure/config/ConfigurationType.java

```diff
@@ -278,40 +305,48 @@ public void setAllPublicFields() {
         removeFields(ConfigurationMemberKind.PUBLIC);
     }
 
-    public boolean haveAllDeclaredMethods() {
-        return allDeclaredMethods;
+    public ConfigurationAccessKind getAllDeclaredMethods() {
+        return allDeclaredMethodsAccess;
     }
 
-    public boolean haveAllPublicMethods() {
-        return allPublicMethods;
+    public ConfigurationAccessKind getAllPublicMethods() {
+        return allPublicMethodsAccess;
     }
```

Comment:
These methods seem to be called only in this class, you should delete them and access the fields directly.

Reply:
Refactored